### ISSUE TITLE
Kommunication set-up

### ISSUE THREAD

**Issue by kkmann** [pharmaR/regulatory-r-repo-wg#20]
How are we ensuring open and transparent communication? We currently, organize everything on github. The advantage is that we have a single place for finding stuff. Need to make sure this is not impeding less git-savy people from participating. Love the idea of meeting minutes with action items as issues; nice an traceable!

- [ ] add contribution guidelines with detailed description of set up and how to get engaged. Link in the README.md, and as a pinned issue / discussion item
- [ ] clarify split between GH discussions and issues (what goes where)
- [ ] how do we make the calls open? If we stick with google, could we make the calls public on GH in advance? Consider jitsi meet https://meet.jit.si/ as open alternative?
- [ ] Do we keep materials  (slides, ...) on GH as well or do we need a file repository? Slight preference for all in this repo to keep it completely open.

**Comment by dgkf** [comment]
> how do we make the calls open? If we stick with google, could we make the calls public on GH in advance? Consider jitsi meet https://meet.jit.si/ as open alternative?

I have no problem finding something more transparent than Google Meet. 

If we want to really embrace open source, I'd jump to using [matrix](https://matrix.org/docs/projects/try-matrix-now/) + [jitsi](https://meet.jit.si/). I've only used matrix chat clients for text communication for other code projects, but I tested out their jitsi integration today and it seemed reasonably straightforward to set up. That could be a nice way to consolidate chat and videoconferencing if we want to go in that direction.

> Do we keep materials (slides, ...) on GH as well or do we need a file repository? Slight preference for all in this repo to keep it completely open.

Either way is fine by me. I had originally thought of this as a primarily code repository, but we can always break that off when we get there and keep this repo for more administrative stuff.

**Comment by kkmann** [comment]
Kill GH discussions and keep it all in good 'ol github issues. All meterials on GH, ideally markdown based.

**Comment by kkmann** [comment]
@dgkf I'd disable the discussion feature entirely just to make sure. I dont't think there is anything that needs to be salvaged from there. Maybe give it until end Jan or so?

**Comment by dgkf** [comment]
Yeah, sounds good. I'll keep it around for a bit in case anyone feels passionately about keeping discussions, but I'm happy to consolidate things. I agree that right now it's ambiguous and we should commit to one or the other.

**Comment by kkmann** [comment]
x) I fear I never made it past the milestones mark. Would have to look into it as well. Maybe just give it a try and see whether it adds value? Has to be done on the pharmaR level, right?

**Comment by dgkf** [comment]
Just a quick follow-up, at #30. We decided that the split between issues and discussions isn't needed if we want to use this repo primarily as a project-tracking space. 

I'm going to transfer all existing discussions to issues and disable the discussions part of the repo.

**Comment by dgkf** [comment]
And just to work toward closing out the action items on here, regarding this one:

> add contribution guidelines with detailed description of set up and how to get engaged. Link in the README.md, and as a pinned issue / discussion item

I say we either put together a crude version for now, or split this off to a separate issue (possibly connected to a website like we were proposing on our #30 call?). What do you think @kkmann?

**Comment by dgkf** [comment]
Alright, all discussions have been moved to issues. I still need to migrate the actual content of the discussions.

**Comment by kkmann** [comment]
@Crosita @dgkf could you maybe sketch up some contribution guidelines? 

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

I don't think there is anything planned in terms of making the calls more open than just getting in touch via email, right? Then we can strike that point as well and close this.
